forms-flow-api/src/formsflow_api/services/form_process_mapper.py

```diff
@@ -460,8 +460,16 @@ def create_form(data, is_designer, **kwargs):  # pylint:disable=too-many-locals
             process_name = (
                 updated_process_name if updated_process_name else process_name
             )
-            # create entry in process with default flow.
-            FormProcessMapperService.create_default_process(process_name)
+            process_data = data.get("processData")
+            process_type = data.get("processType")
+            if process_data and process_type:
+                # Incase of duplicate form we get process data from payload
+                ProcessService.create_process(
+                    process_data, process_type, process_name, process_name
```

Comment:
why we need process_name 2 times ? @auslin-aot

Reply:
create_process need process_key & process name.. In case of duplicate form, form name is kept as process key & name
![image](https://github.com/user-attachments/assets/7aca104c-90f9-4a1a-9cbc-8a33f615d895)
![Uploading image.png…]()

Reply:
ok . thanks